frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx

```diff
@@ -48,7 +49,7 @@ export const FormSelect = ({
 					labelId={`${name}-label`}
 					id={name}
 					label={label}
-					{...otherProps}
+					className={className}
```

Comment:
what happened here? `otherProps`  would also work for `className`

Reply:
Fixed